src/index/ivf/ivf_config.h

```diff
@@ -130,6 +130,40 @@ class IvfSqConfig : public IvfConfig {};
 
 class IvfBinConfig : public IvfConfig {};
 
+class IvfSqCcConfig : public IvfFlatCcConfig {
+ public:
+    // user can use code size to control ivf)sq_cc quntizer type
+    CFG_INT code_size;
```

Comment:
I would not call it `code_size`, it is rather a `quantizer_type`.
`ScalarQuantizer` from Faiss has `QuantizerType qtype;` corresponding name.
Also, there is a typo a line above `ivf)sq_cc`.

Reply:
IVF_SQ_CC will use as a growing index in milvus. Milvus choose quantizer_type by memory uasge.  I think code size is eailer to calculate the predict memory usage.